deploy/resources.go

```diff
@@ -0,0 +1,6 @@
+package deploy
+
+import "embed"
+
+//go:embed templates/notificationtemplates/*
+var Files embed.FS
```

Comment:
minor thing , but maybe we can name this var something more specific `notificationtemplateFS` or something similar ?

Reply:
yes absolutely! that's a great name, Thanks!

Reply:
done in [fb7de08](https://github.com/codeready-toolchain/host-operator/pull/754/commits/fb7de08aadf0e35b6ffd34b707f24526645a3feb)